src/utils/getPaths.js

```diff
@@ -12,6 +12,10 @@ export default function getPaths(context) {
       ? compilation.getPath(compilation.outputOptions.publicPath)
       : '';
 
+    if (publicPath) {
+      context.logger.info(`webpack output is served from "${publicPath === 'auto' ? '/' : publicPath}" URL`);
+    }
```

Comment:
It will be spam on each request, we need move it only on constructor place

Reply:
I will update 👍🏻